src/lib.rs

```diff
@@ -160,12 +159,14 @@ pub fn get() -> usize {
 /// check if the argument matches the current mutation count
 ///
 /// this simplifies operations like `if mutagen::MU.now(42) { .. }`
-pub fn now(n: usize) -> bool {
+pub fn now(n: usize, flag: &AtomicUsize, mask: usize) -> bool {
+    report_coverage(n..(n + 1), flag, mask);
```

Comment:
What about pushing the report to `MU.now(...)` and `MU.diff(...)`? This way we will make sure that future implementors will report the coverage.

Reply:
This would embed the coverage reporting within the mutation code, which may hinder testability. Otherwise I have no preference either way, but this was the easiest way to refactor it.